clayui.com/src/templates/docs.js

```diff
@@ -465,5 +492,8 @@ export const pageQuery = graphql`
 		mdxTab: mdx(fields: {slug: {ne: "", eq: $sibling}}) {
 			body
 		}
+		apiTab: mdx(fields: {slug: {ne: "", eq: $sibling}}) {
+			body
+		}
```

Comment:
One thing to keep in mind, this can help you better understand how it works. `createPages.js` calls this template in each document (.md, .mdx) that it is going through and we receive the values ​​that were passed in that line: https://github.com/liferay/clay/blob/a50d9b724eedfab8fff5585d4a0cf783a7cb8740/clayui.com/src/templates/docs.js#L427-L428

Which are the values ​​coming from the call from `createPage`, which is the `context` here: https://github.com/liferay/clay/blob/a50d9b724eedfab8fff5585d4a0cf783a7cb8740/clayui.com/gatsby/createPages.js#L115-L125

Once we have the `siblings` declared we can get all the documents, in this case we need to get it using `allMdx`, I think we can also kill the above query that gets `mdxTab` and stop using `sibling`, we can only use `siblings` and to differentiate which document belongs to each tab, we add a new value in the frontmatter for this something like:
```markdown
---
tabName: 'API'
---
```

Or we can use the `title` for this case too, I think it makes more sense too. If the document becomes a sibling of another.

And we dynamically created the tabs. The query to get the content of the siblings, I think it would look like this (not tested):
```graphql
allMarkdownRemark(filter: {fields: {slug: {in: $siblings}}}) {
	edges {
		node {
			frontmatter {
				tabName
			}
			body
		}
	}
}
```